mspray/apps/reveal/tests/test_utils.py

```diff
@@ -149,6 +210,113 @@ def test_add_new_point(self):
         add_spray_data(data=input_data)
         self.assertEqual(1, SprayDay.objects.all().count())
 
+    def test_add_new_polygon(self):
+        """
+        Test adding data with new Polygon
+        """
+        SprayDay.objects.all().delete()
```

Comment:
Should this be in a `tearDown()` method so that every test runs it after executing?